synapse/lib/stormtypes.py

```diff
@@ -847,6 +855,9 @@ def __init__(self, valu, path=None):
             'encode': self._methEncode,
         })
 
+    def __len__(self):
```

Comment:
None of these type specific len implementations need to exist now that we're try/excepting in a single len call

Reply:
We do need them, since we still have a split horizon between Prim objects and non-prim objects being passed around the runtime.